nms/app/fbcnms-projects/magmalte/app/views/network/NetworkInfo.js

```diff
@@ -170,7 +170,7 @@ export function NetworkInfoEdit(props: EditProps) {
               onChange={({target}) =>
                 setNetworkInfo({...networkInfo, id: target.value})
               }
-              readOnly={props.networkInfo ? true : false}
+              disabled={props.networkInfo ? true : false}
```

Comment:
Disabled is a more appropriate label for when these fields should be non-editable, plus it prevents the ability to focus the field altogether, as such I've updated the forms to use `disabled` instead of `readOnly`.